applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -766,6 +772,7 @@ gateway_from_jobj(JObj, #resrc{is_emergency=IsEmergency
     EndpointType = wh_json:get_ne_value(<<"endpoint_type">>, JObj, <<"sip">>),
     #gateway{endpoint_type=EndpointType
              ,server=wh_json:get_value(<<"server">>, JObj)
+             ,port=wh_json:get_value(<<"port">>, JObj)
```

Comment:
Shouldn't this use `wh_json:get_integer_value/2` ? Will return either the Port as an integer or 'undefined'.